### PR TITLE
Docs on becoming a listed user and carve out build to dedicated file

### LINKED ISSUE #551: We are looking for customers to be listed as a customer on keda.sh
We are looking for KEDA customers who are interested to be listed as a customer on keda.sh.

If you are, please comment here and provide a logo that we can use.

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [x] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Fixes #551

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,20 +34,16 @@ We are a Cloud Native Computing Foundation (CNCF) sandbox project.
 - [Getting started](#getting-started)
 - [Deploying KEDA](#deploying-keda)
 - [Documentation](#documentation)
+  - [Become a listed KEDA user!](#become-a-listed-keda-user)
```
Comment (zroubalik) [inline README.md:37]: bad intendation

**Review by zroubalik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tomkerkhove) [reply README.md:37]: Generated by this doctoc tool 🤷‍♂️ Happy to change if you want.

Comment (turbaszek) [reply README.md:37]: Using `##` instead of `###` for `Become a listed KEDA user!` section will fix the issue 👍

File: README.md [hunk 2]
```diff
@@ -63,6 +59,10 @@ There are many ways to [deploy KEDA including Helm, Operator Hub and YAML files]
 
 Interested to learn more? Head over to [keda.sh](https://keda.sh).
 
+### Become a listed KEDA user!
```
Comment (turbaszek) [inline README.md:62]:
```suggestion
## Become a listed KEDA user!
```
This probably is not a subsection of Documentation, right?

Comment (tomkerkhove) [reply README.md:62]: This was intentional as this is more documentation related but can make it more important if we want to.

Comment (turbaszek) [reply README.md:62]: Then doc-toc is right here: https://github.com/kedacore/keda/pull/1112#discussion_r484867125

The `### Become a listed KEDA user!` will be subsection of `## Documentation`

Comment (tomkerkhove) [reply README.md:62]: Yes I know, it was intentional :)

Comment (turbaszek) [reply README.md:62]: I just want to say that assuming your intention (which I think is ok) the doctoc tool generated a proper table of content. If we do want to list only "main sections" we may consider changing `--maxlevel` to `1` in pre-commit config for doctoc

Comment (zroubalik) [reply README.md:62]: IMHO `Become a listed KEDA user!` doesn't have to be a subsection of `## Documentation`

But I am ok with both options.

Comment (tomkerkhove) [reply README.md]: Updated with a new proposal

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ For any contribution you make that impacts the behavior or experience of KEDA, p
 
 ## Creating and building a local environment
 
-[Details on setup of a development environment are found on the README](https://github.com/kedacore/keda#building-quick-start-with-visual-studio-code-remote---containers)
+[Details on setup of a development environment are found on the README](https://github.com/kedacore/keda/BUILD)
```
Comment (turbaszek) [inline CONTRIBUTING.md:46]:
```suggestion
[Details on setup of a development environment are found on the README](https://github.com/kedacore/keda/BUILD.md)
```
or `./BUILD.md`

File: README.md [hunk 4]
```diff
@@ -92,128 +77,18 @@ If interested in contributing or participating in the direction of KEDA, you can
 Just want to learn or chat about KEDA? Feel free to join the conversation in
 **[#KEDA](https://kubernetes.slack.com/messages/CKZJ36A5D)** on the **[Kubernetes Slack](https://slack.k8s.io/)**!
 
-## Building: Quick start with [Visual Studio Code Remote - Containers](https://code.visualstudio.com/docs/remote/containers)
+## Become a listed KEDA user!
 
-This helps you pull and build quickly - dev containers launch the project inside a container with all the tooling
-required for a consistent and seamless developer experience.
+We are always happy to [list users](https://keda.sh/community/#users) who run KEDA in production, learn more about it [here](https://github.com/kedacore/keda-docs/blob/master/README.md#become-a-listed-keda-user).
```
Comment (turbaszek) [inline README.md:82]:
```suggestion
We are always happy to [list users](https://keda.sh/community/#users) who run KEDA in production, learn more about it [here](./README.md#become-a-listed-keda-user).
```
As we do below in L90, I think it's better to keep links pointing to the "current" branch

**Comment by zroubalik** [comment]
The Static Check is still failing

**Comment by zroubalik** [comment]
@tomkerkhove DCO 😅

**Comment by tomkerkhove** [comment]
image

**Pull request closed without merging**